apm-sniffer/apm-sdk-plugin/pulsar-common/src/main/java/org/apache/skywalking/apm/plugin/pulsar/common/PulsarConsumerInterceptor.java

```diff
@@ -73,6 +75,9 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
             Tags.MQ_BROKER.set(activeSpan, serviceUrl);
             Tags.MQ_TOPIC.set(activeSpan, consumer.getTopic());
             activeSpan.setPeer(serviceUrl);
+            if (Pulsar.TRACE_MESSAGE_CONTENTS) {
+                Tags.MQ_BODY.set(activeSpan, StringUtil.cut(new String(msg.getData()), Pulsar.MESSAGE_CONTENTS_MAX_LENGTH));
```

Comment:
`msg.getData()` is a byte[]. Is here OK to use the default charset?

Reply:
We used the default encoding for a long time without any issues, but we are using Java and are unsure if other languages can execute properly. This time, I added the UTF-8 character set and it ran normally in our environment